wiki/Tournaments/Template/en.md

```diff
@@ -1,58 +1,63 @@
 ---
 tags:
-- {ABBR} 20{XX}
-- {ABBR}20{XX}
+- {ABBR}
 ---
+
 <!--
 find-replace these curly braces then delete this comment
+{TOURNAMENT_NAME} = tournament name
 {MODE} = game mode
-{ABBR} = abbreviation of the title e.g. MWC
-{XX} = year, last two digits
+{ABBR} = abbreviation of the title followed by its iteration/year occurence i.e. MWC 4K 2018
 {NTH} = installment ordinal number
 {TOPIC_NUM} = forum thread number
 {STATS_LINK} = link to overall statistics (should be Google Drive link)
 you will still to add other items such as tables
 -->
 
-# {MODE} World Cup 20{XX}
+# {TOURNAMENT_NAME}
 
-![{MODE} World Cup 20{XX} logo](logo.png)
+![{ABBR} Logo](logo.jpg)
 
-The **{MODE} World Cup 20{XX}** (***{ABBR} 20{XX}***) is a country-based {MODE} tournament hosted by the [osu! team](/wiki/People/The_Team). It is the {NTH} installment of the {MODE} World Cup.
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the [osu! team](/wiki/People/The_Team). It is the {NTH} installment of the {TOURNAMENT_NAME}.
 
 ## Tournament Schedule
 
 | Event | Timestamp |
-|---:|---|
-| Registration Phase |  |
-| Drawings |  |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 (12 UTC) |
```

Comment:
Idk if this is the default format. I've seen differences in formatting on OWC and MWC.

Reply:
I guess we can go with the recent one.